cg/services/parse_run_completion_status_service/parse_run_completion_status_service.py

```diff
@@ -0,0 +1,37 @@
+"""Service to parse the RunInfo.xml file."""
+
+import re
+from pathlib import Path
+
+from xml.etree.ElementTree import ElementTree
+
+from cg.constants.demultiplexing import RunCompletionStatusNodes
+from cg.io.xml import read_xml
+from datetime import datetime
+
+DATE_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
+
+
+class ParseRunCompletionStatusService:
+
+    @classmethod
+    def format_time(cls, time: str) -> datetime:
```

Comment:
Could this function belong to `utils/time.py`? It looks like a general function independent of this service

Reply:
Moved it